qiskit_experiments/curve_analysis/curve_analysis.py

```diff
@@ -360,14 +363,30 @@ def _default_options(cls) -> Options:
         options.style = PlotterStyle()
         options.extra = dict()
         options.curve_fitter_options = dict()
-
-        # automatically populate initial guess and boundary
-        fit_params = cls._fit_params()
-        options.p0 = {par_name: None for par_name in fit_params}
-        options.bounds = {par_name: None for par_name in fit_params}
+        options.p0 = {}
```

Comment:
Hint for review. This cannot be initialized with parameter names, because fixed params are now instance attribute that user can modify after the instance is created. Note that now we have `.parameters` property, thus user can know what parameters live in the fit model.